src/main/java/com/uu_demo/models/dto/RoleDto.java

```diff
@@ -0,0 +1,18 @@
+package com.uu_demo.models.dto;
+
+
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class RoleDto {
+
+    private Long id;
+
+    private String fullName;
```

Comment:
The term _full name_ relates to the user's personal information (like "John Smith"). If you mean a name of a role here, name it `name`.